static/js/components/ErrorMessage_test.js

```diff
@@ -144,7 +144,7 @@ describe("ErrorInfo", () => {
     });
 
     describe('profile page', () => {
-      it('shows an error when profile get has errored', () => {
+      it('shows an error when profile get has errored instead of leaving the spinner up', () => {
```

Comment:
It doesn't show an error at all, it only starts the request for the profile and asserts that the spinner is shown